ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/platform.cfg

```diff
@@ -5,6 +5,28 @@ transceiver qsfp default-mode 4x10G
 hostname platform
 !
 platform trident forwarding-table partition 2
+platform trident mmu queue profile mc_example_profile apply
+!
+platform trident mmu queue profile mc_example_profile
+    egress multicast queue 0 reserved 0
+    egress multicast queue 0 drop-precedence 1 drop-threshold 3/4
+    egress multicast queue 1 reserved cells 0
+    egress multicast queue 1 threshold 1/64
+    egress multicast queue 7 reserved cells 0
+    egress multicast queue 7 threshold 1/64
+    egress unicast queue 1 reserved bytes 0
+    egress unicast queue 2 reserved cells 0
+    egress unicast queue 2 threshold 1/8
+!
+platform trident mmu queue profile unused_profile
+    egress multicast queue 0 reserved 0
+    egress multicast queue 0 drop-precedence 2 drop-threshold 1
+    egress multicast queue 1 reserved cells 0
+    egress multicast queue 1 threshold 8
+    egress unicast queue 1 reserved bytes 0
+    egress unicast queue 1 drop-precedence 1 drop-threshold 1/8
+    egress unicast queue 2 reserved cells 0
+    egress unicast queue 2 threshold 1/8
 platform sand qos map traffic-class 0 to network-qos 0
 platform sand qos map traffic-class 1 to network-qos 7
 platform sand qos map traffic-class 2 to network-qos 15
```

Comment:
Are we sure the ordering and spacing is similar to EOS here? It looks wrong to me.

Reply:
Tested on a 7050DX and it looks to me similar to what is being presented in the MR. Do you see any specific differences?

![image](https://github.com/aristanetworks/ansible-avd/assets/109683698/3163f980-67bc-4232-b48a-d8f0d71c7782)

Reply:
You need to run without any section commands. You have `!` between each profile. Maybe that is fine - I don't know. Also at the end in the example I commented on, where is no divider to the next section (which may be a different problem).
Also your output here shows that the ordering of multicast and unicast is opposite of the PR.

Reply:
Below is the snippet from show run (no section command).
- the output has ! between each profile
- in reality, would we ever have the trident and sand platform being referenced on the same device? If I add a ! separator to between sand and trident, I need to add logic to make sure we don't get two ! consecutively if there is not `platform trident` section. Shall I do that?
- Agreed with the multicast/unicast ordering and that should be fixed now
- I have already updated the platform command section of the molecule config. Not sure what else to add/test
 
![image](https://github.com/aristanetworks/ansible-avd/assets/109683698/82be0231-fd1d-474e-8da6-a0b918725d3f)

Reply:
This should now be resolved with the refactor of the Jinja template.